cas/incident-report.md

```diff
@@ -228,7 +242,7 @@ In the case of Incident Reports with a Whiteboard field of "revocation-delay", r
 
 #### How are reports closed?
 
-When all Action Items are complete and no outstanding comments or questions remain, CA Owners MUST request closure in a Bugzilla comment using the template [below](#incident-closure-summary). Upon doing so, a final call for comments will be made by a Bugzilla moderator, and the report will be closed accordingly.
+When all Action Items are complete and no outstanding comments or questions remain, CA Owners MUST (1) request closure in a Bugzilla comment using the template [below](#incident-closure-summary) and (2) select the "Request information from" checkbox and add incident-reporting [at] ccadb [dot] org. Upon doing so, a final call for comments will be made by a Bugzilla moderator, and the report will be closed accordingly.
```

Comment:
I think it would be nice to explicitly say something like "After an Incident Closure Summary has been posted, CA Owners are _not_ required to continue updating the incident unless further comments are made."

I'm seeing lots of incidents today where CAs are continuing to post "no updates" comments every 7 days after posting their closure report, and it would be good to reassure them that that's not necessary.

Reply:
Agree, and explicitly stating updates are required if questions are asked:

```suggestion
When all Action Items are complete and no outstanding comments or questions remain, CA Owners MUST (1) request closure in a Bugzilla comment using the template [below](#incident-closure-summary) and (2) select the "Request information from" checkbox and add incident-reporting [at] ccadb [dot] org. Upon doing so, a final call for comments will be made by a Bugzilla moderator, and the report will be closed accordingly. After an Incident Closure Summary has been posted, CA Owners are not required to continue updating the incident report unless further questions are asked.
```

Reply:
This is a good change to make. Would it be possible to also update the normative requirements for update cadence to reflect this?
https://github.com/mozilla/www.ccadb.org/pull/198/files#diff-bf49bf90874e83682c38c07450add072bf7690dd4a9dc10b3f27dec4ab39b5b6R217

Reply:
@CBonnell attempted to address this in https://github.com/mozilla/www.ccadb.org/pull/198/commits/d3ca25fa992cf3c8c98e2bb4295d890b9caac102.